.github/workflows/ci.yml

```
@@ -33,35 +29,53 @@ jobs:
       matrix:
         version: [20, 22]
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
       - name: Using Node.js ${{ matrix.version }}
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v5
         with:
           node-version: ${{ matrix.version }}
       - name: Install dependencies
         run: npm install
       - name: Run tests
         run: npm test
 
-  build_and_publish:
-    needs: [test]
+  build:
     runs-on: ubuntu-latest
+    name: Build
```

The job named `publish` below references `test`.

```suggestion
  test:
    runs-on: ubuntu-latest
    name: Build and test
```